### PR TITLE
GRW-63 Add Web Onboarding DK - Contents + Accident

### PR DESCRIPTION
**Pull request by gustaveen**
GraphQL mutation passed in Angel preview

### CODE REVIEW

**Review by styrelius — COMMENTED** [review]
So in web-onboarding we're referring to this file as `Web Onboarding DK - Danish Contents-Accident` - should we change this one or change in web-onboarding?

image

**Comment by styrelius** [comment]
The naming in web-onboarding is following the same pattern as the naming for the Norwegian files.

File: angel-data/Web Onboarding DK - Contents + Accident.json [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+    {
+      "key": "coInsured",
+      "value": "householdSize - 1"
+    }
+  ],
+  "id": "e99d8925-583c-4226-999b-e049d00e9146",
+  "ifid": "258ED9C4-5F57-475B-9CCE-7CAC4964F094",
+  "keywords": {},
+  "lastUpdate": "2021-03-15T16:10:22.189Z",
+  "name": "Web Onboarding DK - Danish Contents-Accident",
```
Comment (gustaveen) [inline angel-data/Web Onboarding DK - Contents + Accident.json:12]: Yes, I copied the name from `routes.tsx` so it should be the same :) @styrelius

Comment (styrelius) [reply angel-data/Web Onboarding DK - Contents + Accident.json:12]: Ah, sorry, I just saw the name of the file here and thought that's where the name came from 🙈

**Review by styrelius — APPROVED** [review]
Great job! 🇩🇰 ✨

File: angel-data/Web Onboarding DK - Contents + Accident.json (hunk 1 above)
Comment (gustaveen) [reply angel-data/Web Onboarding DK - Contents + Accident.json:12]: No worries, it wasn't super clear :D

**Pull request merged**